public/project/css/nav.css

```diff
@@ -0,0 +1,45 @@
+/* Nav Container */
+.nav-container {
+  height: 70px;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 2;
+}
+
+/* Nav Logo Top Left */
+.nav-container > img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 30px;
+  width: 240px;
+}
+
+/* Hamburger */
+.ion-navicon {
+  color: white;
+  font-size: 40px;
+  right: 33px;
+  top: 24px;
+  transform: scale(1.3, 1);
+}
+
```

Comment:
Cool makes sense .. this element also has a global class (was trying to make re-useable classes) called top-right. So I just replaced top: 24px with padding-top 24px.